show-land/sass/main.scss

```diff
@@ -0,0 +1,6 @@
+@import"variables";
+
+@import"top-nav";
+@import"typography";
+@import"../../shared/scss/components/button";
+@import"../../shared/scss/base/reset";
```

Comment:
Please separate the imports into logical parts like you have done for your variables file